.gitignore

```diff
@@ -20,11 +20,15 @@ hw/ip/power_manager/data/power_manager.hjson
 hw/ip/power_manager/rtl/power_manager.sv
 hw/ip/power_manager/rtl/power_manager_reg_top.sv
 hw/ip/power_manager/rtl/power_manager_reg_pkg.sv
-
+hw/system/pad_control/data/pad_control.hjson
```

Comment:
Did you omit the hw/core-v-mini-mcu/pad_ring.sv on purpose?

Reply:
yes, it should not be part of the files either